roles/lib_utils/library/docker_creds.py

```diff
@@ -131,14 +131,15 @@ def load_config_file(module, dest):
         return {}
 
 
-def gen_skopeo_cmd(registry, username, password, proxy_vars, image_name):
+def gen_skopeo_cmd(registry, username, password, proxy_vars, image_name, tls_verify):
     '''Generate skopeo command to run'''
     skopeo_temp = ("{proxy_vars} timeout 10 skopeo inspect"
                    " {creds} docker://{registry}/{image_name}")
     # this will quote the entire creds argument to account for special chars.
     creds = pipes.quote('--creds={}:{}'.format(username, password))
     skopeo_args = {'proxy_vars': proxy_vars, 'creds': creds,
-                   'registry': registry, 'image_name': image_name}
+                   'registry': registry, 'image_name': image_name,
+                   'tls_verify': tls_verify}
```

Comment:
This will result in just 'True' or 'False' being passed to the command, need to format the command similar to creds.